opentech/apply/activity/messaging.py

```diff
@@ -159,10 +159,10 @@ def process_send(self, message_type, recipients, events, request, user, submissi
 
             if not settings.SEND_MESSAGES:
                 if recipient:
-                    message = '{} [to: {}]: {}'.format(self.adapter_type, recipient, message)
+                    debug_message = '{} [to: {}]: {}'.format(self.adapter_type, recipient, message)
```

Comment:
Avoid messing up the message variable if there is more than one recipient.

Reply:
👍